mobile/src/main/res/layout/fragment_login.xml

```diff
@@ -1,40 +1,40 @@
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingLeft="16dp"
-    android:paddingRight="16dp">
+    android:orientation="vertical"
+    android:padding="40dp">
 
     <ImageView
         android:id="@+id/imageView"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_above="@+id/twitter_button"
-        android:layout_centerHorizontal="true"
-        android:layout_marginBottom="54dp"
+        android:layout_height="0dp"
+        android:layout_weight="3"
+        android:layout_gravity="center_horizontal"
         android:importantForAccessibility="no"
-        android:src="@drawable/placeholder_logo" />
+        android:src="@drawable/logo" />
```

Comment:
In the android developer guide for supporting multiple screens it says only to provide an xxxhdpi resource for the launcher icon, not any other images.

http://developer.android.com/guide/practices/screens_support.html